Firestore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- [feature] Add option to allow SDK create cache indexes automatically to
```

Comment:
Edit:

"[feature] Add the option to allow the SDK to create cache indexes automatically to
  improve query execution locally."